area.py

```diff
@@ -203,12 +235,13 @@ def get_all_bodies(self):
         return bodies
 
     def get_body_radius(self, body):
-        if body.name == BodyName.SUN:
-            return get_sun_scale_radius(self.width, self.height, self.zoom)
+            if body.name == BodyName.SUN:
+                return get_sun_scale_radius(self.width, self.height, self.zoom)
+
+            else:
+                return get_planet_scale_radius(
+                    self.width, self.height, body, self.zoom)
```

Comment:
Why the extra indentation?

Reply:
i corrected it also on Zed new IDE its happening sorry for it you can check latest.